packages/config-loader/src/lib/transform/include.test.ts

```diff
@@ -14,25 +14,54 @@
  * limitations under the License.
  */
 
+import { JsonValue } from '@backstage/config';
 import * as os from 'os';
 import { resolve as resolvePath } from 'path';
 import { createIncludeTransform } from './include';
 
 const root = os.platform() === 'win32' ? 'C:\\' : '/';
+const substituteMe = '${MY_SUBSTITUTION}';
+const mySubstitution = 'fooSubstitution';
 
 const env = jest.fn(async (name: string) => {
   return ({
     SECRET: 'my-secret',
   } as { [name: string]: string })[name];
 });
 
+const substitute = async (
```

Comment:
Could prolly slim down the typing here using `const substitute: TransformFunc = ...`

Reply:
Done! 👍🏻   Sorry for the noise, not 100% with my Typescript just yet.